var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -57,12 +59,16 @@ class Gromacs(CMakePackage):
     version('4.6.7', sha256='6afb1837e363192043de34b188ca3cf83db6bd189601f2001a1fc5b0b2a214d9')
     version('4.5.5', sha256='e0605e4810b0d552a8761fef5540c545beeaf85893f4a6e21df9905a33f871ba')
 
-    variant('mpi', default=True, description='Activate MPI support (disable for Thread-MPI support)')
+    variant('parallel', default='mpi',
+            values=('thread-mpi', 'mpi', 'all'),
+            multi=True,
+            description='parallelization mode (Thread-MPI or MPI support)')
     variant('shared', default=True,
             description='Enables the build of shared libraries')
-    variant(
-        'double', default=False,
-        description='Produces a double precision version of the executables')
+    variant('precision', default='single',
+            values=('single', 'double', 'all'),
+            multi=True,
+            description='Floating poing precision of the executables')
```

Comment:
This doesn't make sense to me as a multi-valued variant. I think we should either have it `multi=False` or remove the value `all`.

Reply:
Good catch, I updated the commit message but forgot to remove this from the code (!)